scripts/linters/pylint_extensions_test.py

```diff
@@ -3059,29 +3063,32 @@ def test_disallowed_removals_str(self):
         message_remove_example_func = testutils.Message(
             msg_id='remove-disallowed-function-calls',
             node=call1,
-            args=b'example_func'
+            args='example_func',
+            confidence=interfaces.UNDEFINED
```

Comment:
Why are we passing this new 'confidence' parameter everywhere?

Reply:
In py3 the pylint newly uses the confidence argument in some places, we use these vars to compare the output of pylint with our assumptions and so that we need to add the confidence arg.